.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+public
```

Comment:
I add `public` on `.gitignore` because this is has generated by `script` like `yarn dev`. 
this is correct if i removed ? @munirapp